pkg/autoscaler/autoscaler/autoscaler_manager.go

```diff
@@ -55,14 +56,15 @@ func NewAutoScalerManager(
 	kubeInformerFactory kubeinformers.SharedInformerFactory,
 	recorder record.EventRecorder) *autoScalerManager {
 	tcLister := informerFactory.Pingcap().V1alpha1().TidbClusters().Lister()
-	stsLister := kubeInformerFactory.Apps().V1().StatefulSets().Lister()
 	return &autoScalerManager{
 		kubecli:   kubecli,
 		cli:       cli,
 		tcLister:  tcLister,
```

Comment:
How about
```suggestion
		tcLister:  informerFactory.Pingcap().V1alpha1().TidbClusters().Lister(),
```

Reply:
When creating `tcControl`, we also need `tcLister` and I think creating one is enough